### DISCUSSION TITLE
Aggregation of current discussions about feed + Allow each user to pick type of feed they want

### DISCUSSION THREAD

**Discussion by shailshouryya** [community/community#66226] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

While the new algorithmic feed might be useful for people who want a more social media-esque type of feed, many people find the new feed (at best) overwhelming or even useless:

- https://github.com/orgs/community/discussions/66398
- https://github.com/orgs/community/discussions/66391
- https://github.com/orgs/community/discussions/66378
- https://github.com/orgs/community/discussions/66376
- https://github.com/orgs/community/discussions/66363 (events for personal repos)
- https://github.com/orgs/community/discussions/66354
- https://github.com/orgs/community/discussions/66352
- https://github.com/orgs/community/discussions/66347
- https://github.com/orgs/community/discussions/66345 (commits discussion)
- https://github.com/orgs/community/discussions/66344
- https://github.com/orgs/community/discussions/66342
- https://github.com/orgs/community/discussions/66336
- https://github.com/orgs/community/discussions/66334
- https://github.com/orgs/community/discussions/66333
- https://github.com/orgs/community/discussions/66331
- https://github.com/orgs/community/discussions/66329
- https://github.com/orgs/community/discussions/66328
- https://github.com/orgs/community/discussions/66320
- https://github.com/orgs/community/discussions/66319
- https://github.com/orgs/community/discussions/66318
- https://github.com/orgs/community/discussions/66316
- https://github.com/orgs/community/discussions/66310
- https://github.com/orgs/community/discussions/66305
- https://github.com/orgs/community/discussions/66301
- https://github.com/orgs/community/discussions/66298
- https://github.com/orgs/community/discussions/66290
- https://github.com/orgs/community/discussions/66286
- https://github.com/orgs/community/discussions/66285
- https://github.com/orgs/community/discussions/66281
- https://github.com/orgs/community/discussions/66276
- https://github.com/orgs/community/discussions/66274
- https://github.com/orgs/community/discussions/66273
- https://github.com/orgs/community/discussions/66272
- https://github.com/orgs/community/discussions/66271
- https://github.com/orgs/community/discussions/66268
- https://github.com/orgs/community/discussions/66267
- https://github.com/orgs/community/discussions/66264
- https://github.com/orgs/community/discussions/66263
- https://github.com/orgs/community/discussions/66261
- https://github.com/orgs/community/discussions/66260
- https://github.com/orgs/community/discussions/66259 (formatting https://github.com/dashboard-feed)
- https://github.com/orgs/community/discussions/66256
- https://github.com/orgs/community/discussions/66253
- https://github.com/orgs/community/discussions/66252
- https://github.com/orgs/community/discussions/66251
- https://github.com/orgs/community/discussions/66249 (poll for building independent feed)
- https://github.com/orgs/community/discussions/66247
- https://github.com/orgs/community/discussions/66246
- https://github.com/orgs/community/discussions/66244
- https://github.com/orgs/community/discussions/66242
- https://github.com/orgs/community/discussions/66241
- https://github.com/orgs/community/discussions/66240
- https://github.com/orgs/community/discussions/66238
- https://github.com/orgs/community/discussions/66235
- https://github.com/orgs/community/discussions/66229
- https://github.com/orgs/community/discussions/66228
- https://github.com/orgs/community/discussions/66226 (this one)
- https://github.com/orgs/community/discussions/66224
- https://github.com/orgs/community/discussions/66223
- https://github.com/orgs/community/discussions/66222
- https://github.com/orgs/community/discussions/66221
- https://github.com/orgs/community/discussions/66219
- https://github.com/orgs/community/discussions/66218
- https://github.com/orgs/community/discussions/66217
- https://github.com/orgs/community/discussions/66215
- https://github.com/orgs/community/discussions/66211
- https://github.com/orgs/community/discussions/66210
- https://github.com/orgs/community/discussions/66209
- https://github.com/orgs/community/discussions/66208
- https://github.com/orgs/community/discussions/66207
- https://github.com/orgs/community/discussions/66205
- https://github.com/orgs/community/discussions/66204
- https://github.com/orgs/community/discussions/66203
- https://github.com/orgs/community/discussions/66201
- https://github.com/orgs/community/discussions/66200
- https://github.com/orgs/community/discussions/66199
- https://github.com/orgs/community/discussions/66196
- https://github.com/orgs/community/discussions/66195
- https://github.com/orgs/community/discussions/66193
- https://github.com/orgs/community/discussions/66192
- https://github.com/orgs/community/discussions/66189
- https://github.com/orgs/community/discussions/66188
- https://github.com/orgs/community/discussions/66187
- https://github.com/orgs/community/discussions/66185
- https://github.com/orgs/community/discussions/66184
- https://github.com/orgs/community/discussions/66183
- https://github.com/orgs/community/discussions/66182
- https://github.com/orgs/community/discussions/66181
- https://github.com/orgs/community/discussions/66180
- https://github.com/orgs/community/discussions/66179
- https://github.com/orgs/community/discussions/66178
- https://github.com/orgs/community/discussions/66176
- https://github.com/orgs/community/discussions/66175
- https://github.com/orgs/community/discussions/66173
- https://github.com/orgs/community/discussions/66172
- https://github.com/orgs/community/discussions/66171
- https://github.com/orgs/community/discussions/66169
- https://github.com/orgs/community/discussions/66167
- https://github.com/orgs/community/discussions/66166
- https://github.com/orgs/community/discussions/66165
- https://github.com/orgs/community/discussions/66162
- https://github.com/orgs/community/discussions/66159
- https://github.com/orgs/community/discussions/66158
- https://github.com/orgs/community/discussions/66156
- https://github.com/orgs/community/discussions/66155
- https://github.com/orgs/community/discussions/66154
- https://github.com/orgs/community/discussions/66133
- https://github.com/orgs/community/discussions/65343 (official discussion)
- https://github.com/orgs/community/discussions/13130 (meta discussion)

Adding a button to toggle the type of feed a user wants would be very helpful - similar to the way it was before this new feed rollout.

**Comment by shailshouryya** [comment]
To filter by newest posts (and not by recent comment activity default), you need to go to https://github.com/orgs/community/discussions?discussions_q=is%3Aopen+sort%3Adate_created

**Reply by shailshouryya** [reply]
See https://github.com/orgs/community/discussions/categories/feed?discussions_q=is%3Aopen+category%3AFeed+sort%3Adate_created to filter by the `Feed` category

**Reply by shailshouryya** [reply]
Slightly tangential, but keep in mind that upvotes will not bump a discussion to the top of the discussions (if you're sorting by `Latest activity`, which is the default): https://github.com/orgs/community/discussions/66231

**Comment by beeman** [comment]
Someone posted a link to my topic on HackerNews! An upvote will help visibility  https://news.ycombinator.com/item?id=37413126

**Comment by shailshouryya** [comment]
Looks like the "official" discussion has been updated recently: https://github.com/orgs/community/discussions/65343

**Comment by ettaboyle** [comment]
_We've added the following update to the [Updates to your GitHub Feed](https://github.com/orgs/community/discussions/65343#top) Discussion Announcement originally posted August 29, 2003._


## _Sept. 12th Update_
_With the latest feed update on September 6, 2023, we made changes to the underlying technology of the feed to improve overall platform performance. As a result, we removed the functionality for “push events for repositories a user is subscribed to”. We don’t take these changes lightly, but as our community continues to grow tremendously, we have to prioritize our availability, user experience, and performance._ 


_Thanks to feedback from the community, we have updated our [changelog](https://github.blog/changelog/2023-09-06-updates-to-your-github-feed/) to clarify and have fixed the following bugs from the initial September 6th release:_


- _We were showing releases for organizations that you follow, instead of only showing them for repositories that you watch or star. This is now fixed._ 
- _We have also addressed a problem where 'Followed You' cards were sometimes appearing out of chronological order._
- _We do not plan to re-include “push events for repositories a user is subscribed to” for the reasons listed above, but we will continue to address feedback as it aligns with our platform goals._


_We understand that many of you are upset with the recent changes to your feed. We should have done a better job communicating recent changes and how those decisions relate to our broader platform goals. Your continued feedback is invaluable as we evolve and continue to strive to provide a first-class developer experience that helps every developer be happier and more productive._
